airgap/terraform/main.tf

```diff
@@ -240,6 +252,8 @@ resource "aws_instance" "lh_registry_aws_instance" {
 
   tags = {
     Name = "lh-registry-${random_string.random_suffix.id}"
+    DoNotDelete = "true"
```

Comment:
Do we need to have `DoNotDelete` to all resourecs as well? Is this tag really taking effect?

Reply:
As far as I know, `DoNotDelete` prevent ec2 instances from being deleted by rancher IT, but it's not applicable for other resources like vpc, subnet, etc. Other resources will live forever without being periodically deleted even though they don't have any tag.

Reply:
So there would be bunch of orphaned resources without associated to any ec2 instances, since sometimes the ec2 instance is deleted manually but other resources associated to this ec2 instance aren't cleaned up.
Currently https://github.com/longhorn/longhorn-tests/pull/1116 only deletes long-running ec2 instances with their associated resources, but if ec2 instance is manually deleted, the remaining orphaned resources are still unable to be cleaned up. The next step of https://github.com/longhorn/longhorn/issues/3913 should be reverse lookup these resources to find the associated ec2 instances, if the associated ec2 instances not found, it means the resource is orphaned and should be cleaned up.

Reply:
yeah, sounds good to me.